common/graphics/persistent_cache.h

```diff
@@ -45,7 +45,26 @@ class PersistentCache : public GrContextOptions::PersistentCache {
   //
   // This is used to specify persistent cache filenames and service protocol
   // json keys.
-  static std::string SkKeyToFilePath(const SkData& data);
+  static std::string SkKeyToFilePath(const SkData& key);
+
+  // Allocate a MallocMapping containing the given key and value in the file
+  // format used by the cache.
+  static std::unique_ptr<fml::MallocMapping> BuildCacheObject(
+      const SkData& key,
+      const SkData& data);
+
+  // Header written into the files used to store cached Skia objects.
+  struct CacheObjectHeader {
+    // A prefix used to identify the cache object file format.
+    static const uint32_t kSignature = 0xA869593F;
+    static const uint32_t kVersion1 = 1;
+
+    CacheObjectHeader(uint32_t _key_size) : key_size(_key_size) {}
```

Comment:
Minor nit: The underscore position is not conventional. Perhaps `p_key_size`?

Reply:
done